pom.xml

```diff
@@ -4880,6 +4880,7 @@
         <groupId>org.nuxeo.client</groupId>
         <artifactId>nuxeo-java-client</artifactId>
         <version>${nuxeo.java.client.version}</version>
+        <scope>test</scope>
```

Comment:
:-1:  nuxeo-java-client is not a "testing" library, why setting it with scope test by default? Thus needing an explicit scope compile in nuxeo-distribution/nuxeo-functional-tests/pom.xml and what about the possible usage outside? That breaks POM compliance.

This would be an important change, not mentioned in the JIRA ticket, out of purpose of the ticket.

Reply:
Because the platform hasn't to depend on nuxeo-java-client on compile `scope`, it was a forgetting at the beginning. The only usage is in `nuxeo-functional-tests` which is a test library requiring a dependency which doesn't have to belong to compile scope. No other modules require it as it's the client of platform. If we want to use it in the platform, we have to only use it in tests.

I don't see why it breaks the POM compliance ?
But sorry, if I need to create a dedicated tickets for two lignes which fix a little thing for a dependency (wrongly scoped) used in one module which uses it in one class (perhaps 2), I'll remove these lines.
What do you think ?

Reply:
I removed the two lines.